### PR TITLE
Add format verification for tests, and apply it

### LINKED ISSUE #595: Add format verification for tests
See comment https://github.com/jaegertracing/jaeger-operator/pull/588#discussion_r313268210

### PR DESCRIPTION
**Pull request by kevinearls**
Signed-off-by: Kevin Earls <kearls@redhat.com>

This fixes #595

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,7 +21,7 @@ ES_OPERATOR_IMAGE ?= quay.io/openshift/origin-elasticsearch-operator:4.1
 SDK_VERSION=v0.8.1
 
 LD_FLAGS ?= "-X $(VERSION_PKG).version=$(OPERATOR_VERSION) -X $(VERSION_PKG).buildDate=$(VERSION_DATE) -X $(VERSION_PKG).defaultJaeger=$(JAEGER_VERSION)"
-PACKAGES := $(shell go list ./cmd/... ./pkg/... |  grep -v elasticsearch/v1)
+PACKAGES := $(shell go list ./cmd/... ./pkg/...  ./test/... |  grep -v elasticsearch/v1)
```
Comment (objectiser) [inline Makefile]: This is causing the CI failure, as it results in the unit tests being performed including the e2e tests.

**Comment by objectiser** [comment]
@kevinearls looks like the examples tests are failing: https://travis-ci.org/jaegertracing/jaeger-operator/jobs/571881000 - have restarted and still failing.

**Comment by kevinearls** [comment]
@objectiser I restarted the tests and they have passed.  The example tests have failed several times recently, possibly because they run for so long.  I can look into this next week.  It might be worth splitting them into 2 parts, or at least increasing the timeout so Travis doesn't abort them.

**Comment by pavolloffay** [comment]
I have also notived example tests to fail on travis due to timeout. Maybe travis is killing the job because there are not logs in the output for a long time.

**Comment by objectiser** [comment]
@kevinearls Yeah would be good if you could look at #597 when back.

**Comment by kevinearls** [comment]
@objectiser Can you merge this and then I'll start on #597?

**Comment by objectiser** [comment]
@kevinearls done

**Pull request merged**